src/components/Login.js

```diff
@@ -16,6 +16,13 @@ export default function Login(){
     const { setUserdata } = useContext(UserContext);
     const navigate = useNavigate();
 
+    const userlogin = localStorage.getItem('data');
+
+    if(userlogin !== null){
+        navigate("/hoje");
+        setUserdata(JSON.parse(userlogin));
```

Comment:
**suggestion**: it would make sense to change the order of these lines

Reply:
Ok